api/types/client.go

```diff
@@ -339,3 +340,12 @@ type PluginInstallOptions struct {
 	AcceptPermissionsFunc func(PluginPrivileges) (bool, error)
 	Args                  []string
 }
+
+// SecretRequestOption is a type for requesting secrets
+type SecretRequestOption struct {
```

Comment:
This type seems to repeat lot of the `SecretReferenceSpec` information. Is this necessary?